src/LocalStorage/Card.ts

```diff
@@ -0,0 +1,113 @@
+import { assert } from 'console';
+
+export type CardJSON = {'type':string, 'colors':string[], 'name':string, 'value'?:number};

```

Comment:
Hier vielleicht ein Interface benutzen?

Reply:
Ich würde ebenfalls empfehlen, dass wir mal das package Zod verwenden. Das wird in vielen großen Projekten verwendet und hilft sehr Daten zu validieren... Dazu einfach ein Zod-Schema erstellen und danach kann ein Objekt darauf mit parse umgewandelt werden... Und man kann